README.md

```diff
@@ -134,5 +134,48 @@ You can run a few different types of automated test scenarios.
 * Unable to set surface capabilities
 * Selecting an item for a ListSelect and CarouselSelect do not work
 
+## Interactive mode
+
+You can use an "Interactive mode" to test your actions. This mode allows you to communicate with your actions as
+like the Actions Simulator easily.
+
+If you have already installed this library by the steps described on "How to get started" section, the
+`action-interactive` command has been installed at the same time. You can use the command right now, if you have
+installed this library for global. Otherwise, you should be able to find the command at `./node_modues/.bin`
+directory.
+
+When you want to test your actions using the command, run the following:
+
+```bash
+$ action-interactive -l <LOCALE> -c <CREDENTIAL_FILE> "<YOUR_ACTION_NAME>" "<PROMPT>"
+```
+
+* `<YOUR_ACTION_NAME` - The name to invoke your action. If omitted, the default app name for test app is used (ex. "my test app").
+* `<PROMPT>` - The additional prompt you want to pass to your action. Optional.
+* `<LOCALE>` - The locale string. If omitted, "en-US" is used.
```

Comment:
Is this and `credential file` also optional?

Reply:
Yes, both are optional. I intend to add the `Optional.` to them.

Reply:
Well, all items can be omitted. If all omitted, it means the following:

```
action-interactive -l en-US -c ./credentials.json "my test app"
```

I intend to add the description above to the README.md file.